apps/api/v2/src/modules/tokens/tokens.repository.ts

```diff
@@ -30,22 +30,37 @@ export class TokensRepository {
     });
   }
 
+  async getAuthorizationTokenByClientUserIds(clientId: string, userId: number) {
+    return this.dbRead.prisma.platformAuthorizationToken.findFirst({
+      where: {
+        platformOAuthClientId: clientId,
+        userId: userId,
+      },
+    });
+  }
+
   async createOAuthTokens(clientId: string, ownerId: number) {
     const accessExpiry = DateTime.now().plus({ days: 1 }).startOf("day").toJSDate();
     const refreshExpiry = DateTime.now().plus({ year: 1 }).startOf("day").toJSDate();
 
+    const issuedAtTime = Math.floor(Date.now() / 1000);
```

Comment:
great Idea, why not just use Date.now() directly ?

Reply:
/ 1000 to have it in seconds as required by jwt specification

Reply:
will create global jwt service in follow up PR